switcher.js

```diff
@@ -133,10 +141,29 @@ class Switcher {
     return 'bad';
   }
 
+  /**
+   * Checks that the config file has valid data
+   *
+   * @returns {String} - Status of the file's contents
+   */
+  _checkConfigFile() {
+    if (!fs.existsSync(`${this.awsDir}/config`)) {
+      return 'bad';
+    }
+    const contents = fs.readFileSync(`${this.awsDir}/config`, 'utf8');
+    if (contents === '') {
+      return 'empty';
+    }
+    const regexMatch = contents.match(/\[.*\]\n* *output *= *.*\n *region *= *.*/);
```

Comment:
According to [this](https://docs.aws.amazon.com/cli/latest/topic/config-vars.html) the config file may have nested values, and additional values beyond just the region and output, like so:

```
[profile testing]
aws_access_key_id = foo
aws_secret_access_key = bar
region = us-west-2
s3 =
  max_concurrent_requests=10
  max_queue_size=1000
```

This would cause the regex here to [fail to match](https://regex101.com/r/LkHUmp/1) the file, even though the format of the file is good.